api/migrations/functions/createWorkspaceSchema.sql

```diff
@@ -147,16 +147,38 @@ BEGIN
     "workspaceId" uuid NOT NULL DEFAULT ''%s'',
 
     CONSTRAINT "PK_datasource" PRIMARY KEY (id),
-    CONSTRAINT "UNQ_ds_slug" UNIQUE (slug, "workspaceId"),
-    CONSTRAINT "CHECK_datasource_client"
-      CHECK (client = ANY (ARRAY[''sqlite3''::text, ''pg''::text])),
-    CONSTRAINT "CHECK_datasource_type"
-      CHECK (client = ANY (ARRAY[''remote''::text, ''local''::text]))
+    CONSTRAINT "UNQ_ds_slug" UNIQUE (slug, "workspaceId")
 
-  ) INHERITS ("core"."lck_datasource")', $1);
+  ) INHERITS ("core"."lck_datasource")', workspace_id);
 
   -- CREATE all tables for metamodel
 
+  CREATE TABLE "migration" (
```

Comment:
Do you mean, create the `migration` table in the `core` schema and inherit in the dedicated workspace schema ?
If so, for what use cases do you think it could be useful ?

Reply:
Nope, not in core.
I think, I miss some informations. Do you create a table "migration" for each workspace?

Reply:
Well, yes, each dedicated schema for a workspace will have its table `migration`.
This is the way I think we'll be able to undo/redo migration, but also serialize them if we want - one day ! - make templates for a workspace. In this last case, we could have template versions, with migrations managed for each version.